tests/api-diff/modified_variant_type_tests.t

```diff
@@ -195,10 +195,10 @@ Run the api-watcher on the two cmi files
   $ api-diff shapes2.cmi add_component.cmi
   diff module Add_component:
    type shape =
-  -  | Circle of point * int
-  +  | Circle of point * int * int
-  -  | Rectangle of point * point
-  +  | Rectangle of point * point * int
+  -  | Circle of float * float * int
+  +  | Circle of float * float * int * int
```

Comment:
we're missing brackets here and this makes the overall expression incorrect! Also, since `point` is unchanged, I'd expect our code to produce `Same point` rather than `Same foat * float` right?